go/vt/vterrors/constants.go

```diff
@@ -38,3 +42,12 @@ const (
 	// PrimaryVindexNotSet is the error message to be used when there is no primary vindex found on a table
 	PrimaryVindexNotSet = "table '%s' does not have a primary vindex"
 )
+
+const NoReplicationStatusStr = "no replication status"
```

Comment:
Can you group all the `const` together in one block?

Reply:
fixed